NAMtoOWIRamp.pl

```diff
@@ -535,7 +390,7 @@ sub rotateAndFormat {
 ################################################################################
 sub addToFort22 {
     open( F22, ">>$fort22" )
-      || die "ERROR: NAMtoOWIRamp.pl: Failed to open OWI (NWS12) fort.22 file to append a comment line with the met time increment.";
+      || die "ERROR: $this: Failed to open OWI (NWS12) fort.22 file to append a comment line with the met time increment.";
```

Comment:
You can use `$0` here

Reply:
Thank you, I have made those changes as suggested.